draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -10468,9 +10468,11 @@ Content-Type: text/plain
    related to HTTP semantics are about securing server-side applications (code
    behind the HTTP interface), securing user agent processing of content
    received via HTTP, or secure use of the Internet in general, rather than
-   security of the protocol. Various organizations maintain topical
-   information and links to current research on Web application security
-   (e.g., <xref target="OWASP"/>).
+   security of the protocol. URIs are fundamental to HTTP operation; the
+   security considerations of URIs are discussed in
+   <xref target="URI" x:fmt="," x:sec="7"/>. Various organizations maintain
+   topical information and links to current research on Web application
+   security (e.g., <xref target="OWASP"/>).
```

Comment:
I think it would read better as
```suggestion
   security of the protocol. The security considerations for URIs, which
   are fundamental to HTTP operation, are discussed in
   <xref target="URI" x:fmt="," x:sec="7"/>. Various organizations maintain
   topical information and links to current research on Web application
   security (e.g., <xref target="OWASP"/>).
```

Reply:
Leading with "security considerations" does feel more parallel to the previous sentences.
There is perhaps a theoretical ambiguity in this phrasing as to whether it is URIs that are fundamental to HTTP or the security considerations thereof ... which suggests the even shorter "The security considerations for URIs are discussed in [ref]".